Web_Development/Javascript/Topics/5.Syntax/1.Data_Types/README.md

```diff
@@ -1 +0,0 @@
-# To Do
```

Comment:
This shouldn't have been done. Please rename the file `1.Data_Types_In_JavaScript.md ` to `README.md`.